lib/Admin/tiles/image.php

```diff
@@ -64,9 +56,8 @@ class="button add-to-library"
 						data-id="<?php echo esc_attr( $image_id ); ?>"
 						data-filename="<?php echo esc_attr( $image_filename ); ?>"
 						data-description="<?php echo esc_attr( $description ); ?>"
-						data-url="<?php echo esc_attr( $original_url ); ?>"
-						data-thumbnail-url="<?php echo esc_attr( $thumbnail_url ); ?>"
-						data-pager-url="<?php echo esc_attr( $pager_url ); ?>"
```

Comment:
We were previously passing in specific sizes returned by the API. This is no longer needed since we are leveraging the "templated" image url from Widen.